editor/components/block-list/breadcrumb.js

```diff
@@ -56,27 +55,24 @@ export class BlockBreadcrumb extends Component {
 	render( ) {
 		const { uid, rootUID, selectRootBlock, isHidden } = this.props;
 		const { isFocused } = this.state;
-		const selectParentLabel = __( 'Select parent block' );
 
 		return (
-			<NavigableToolbar className={ classnames( 'editor-block-list__breadcrumb', {
+			<div className={ classnames( 'editor-block-list__breadcrumb', {
```

Comment:
Instead of using a wrapping div can we pass the className directly in toolbar component? (Toolbar accepts a className prop).

Reply:
@jorgefilipecosta I've tried, but there's a lot of CSS involving also the block contextual toolbar. I can't make the positioning on desktop/mobile work nicely unless completely refactoring the CSS, which I'd like to avoid.

Worth noting with "wide" images, the breadcrumb completely hides the block movers (happens also on master) so I guess the breadcrumb styling will need some adjustments anyway:

<img width="642" alt="screen shot 2018-05-01 at 10 26 38" src="https://user-images.githubusercontent.com/1682452/39466738-b2444256-4d2a-11e8-80fd-19cfae24f596.png">